docs/REST API/rest-api.md

```diff
@@ -8298,6 +8315,13 @@ _Available in Fleet Premium_
         "host_batch_size": 0
       }
     },
+    "integrations": {
+      "google_calendar": {
+        "enable_calendar_events": true,
+        "email": "name@example.com",
+        "policy_ids": [1, 2, 3]
```

Comment:
@jacobshandling This is out of date. It is:
```
  "policies": [{"name": "policy 1", "id": 1}, {"name": "policy 2", "id": 2}]
```

The names are validated and ids are set when config is saved. When saving, only names are needed.

This means the names and/or ids may not match the actual policies you're seeing.
- name may have been renamed
- id may be gone (deleted)

Try to match on names, so that we are consistent with how we will set configs via fleetctl apply/gitops.

Reply:
@rachaelshaw can we update this according to @getvictor 's comment so this PR is the source of truth?

Reply:
It is possible to support both. Maybe give priority to names. If name is blank, then use the id.